### DISCUSSION TITLE
【Seeking Help】 When modify annotations templation to delete extra space, Zotlit cannot create or update any notes

### DISCUSSION THREAD

**Discussion by Lenthking** [aidenlx/obsidian-zotero#193] [Help] [answered]
When I change the annotation templation like 
[!note] Page <%= it.pageLabel %>
<%= it.imgEmbed %><%= it.text**.replace(/\s*/g,"")** %>

Zotlit cannot work!
Anyway, Zotlit is a very excited add-on, I really really appreciate it!
Thanks!

**Comment by aidenlx** [comment, marked as answer]
You should get rid of `*` within `<%= ... %>` tag

```
[!note] Page <%= it.pageLabel %>
<%= it.imgEmbed %><%= it.text.replace(/\s*/g,"") %>
```

**Comment by aidenlx** [comment]
Answered above at the time — the `**` was Markdown bold leaking into the tag.

For anyone landing here from a search: v2's default template language is Liquid, where the same edit is a filter rather than a JavaScript call, so there is no expression to break:

```liquid
{{ zt.text | replace: " ", "" }}
```

v2 beta is available now: https://zotlit.aidenlx.site/docs/install-zotlit
Migration guide: https://zotlit.aidenlx.site/docs/how-to/migrate-from-v1
